src/prompts/ddl/deeplink-button.tsx

```diff
@@ -0,0 +1,50 @@
+import { Component, h, createRef, RefObject } from 'preact';
+import Clipboard from 'clipboard';
+
+interface DeeplinkButtonProps {
+    style: h.JSX.CSSProperties;
+    class: string;
+    text: string;
+    linkUrl: string;
+    onAction: () => void;
+}
+
+export default class DeeplinkButton extends Component<
+    DeeplinkButtonProps,
+    never
+> {
+    private readonly btnRef: RefObject<HTMLButtonElement>;
+    private clipboard?: Clipboard;
+
+    constructor(props: DeeplinkButtonProps) {
+        super(props);
+
+        this.btnRef = createRef<HTMLButtonElement>();
+    }
+
+    onTouchEnd = () => {
+        this.clipboard = new Clipboard(this.btnRef.current!);
+        this.clipboard.on('success', e => this.props.onAction());
```

Comment:
Can/should we just set this up in compDidMount and let the clipboard class take care of the tap?

Reply:
Yeah, i think I started off with exactly that way of doing it from the lifecycle fn - but it was just not ending up in the clipboard at all - will take another look, might have just been something

Reply:
No longer relevant as the clipboard copy impl has been updated to use a different library and integration.